src/main/scala/backends/CppRunnable.scala

```diff
@@ -23,7 +23,7 @@ private class CppRunnable extends CppLike {
     case _:TIndex | _:TStaticInt => text("int")
     case TSizedInt(_, un) => text(if (un) "unsigned int" else "int")
     case _:TFloat => text("float")
-    case _:TDouble => text("double")
+    case _:TDouble | _:TRational | _:TFixed => text("double")
```

Comment:
Codegen should probably never need to give a type to rationals, right? (This would only happen if we needed to declare a variable or a function argument with the rational type, which I think we rule out.) So to stay defensive, perhaps we should throw an error instead of just producing `double`.

Reply:
Done